### PR TITLE
Scheduling — Display full course details in Schedule View

### PR DESCRIPTION
**Pull request by tygentry**
Altered ScheduleEvent blocks to display the following in this order:

Course title
Department Level - Section #
CRN

With placeholder text for Location and Professor(s) once that data is pulled from the csv.

### CODE REVIEW

File: src/web/src/components/ScheduleEvent.vue [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
   <div class="schedule-event" :style="data.style">
     <div class="event-text">
-      {{ props.crn }} - {{ props.section }}
+      {{ props.title }}
       <br>
+      {{ props.name }} - {{ props.section }}
+      <br>
+      {{ props.crn }}
+      <br>
+      Location
```
Comment (marchdan) [inline src/web/src/components/ScheduleEvent.vue:10]: this part should get the actual course location if it is available

File: src/web/src/components/ScheduleEvent.vue [hunk 2]
```diff
@@ -1,8 +1,17 @@
...
+      <!-- location -->
+      <br>
+      Professor
```
Comment (marchdan) [inline src/web/src/components/ScheduleEvent.vue:13]: this part should get the actual course professor if it is available

**Comment by marchdan** [comment]
#72 Should be helpful with this

**Comment by jshom** [comment]
Screen Shot 2020-04-07 at 7 08 23 PM
1. little packed, can we shrink the size
2. ditto @marchdan, lets remove location and professor for now.

then good for merge

File: src/web/src/components/ScheduleEvent.vue (hunk 1 above)
Comment (tygentry) [reply src/web/src/components/ScheduleEvent.vue:10]: Removed until data is available

File: src/web/src/components/ScheduleEvent.vue (hunk 2 above)
Comment (tygentry) [reply src/web/src/components/ScheduleEvent.vue:13]: Removed until data is available

File: src/web/src/components/ScheduleEvent.vue (hunk 1 above)
Comment (marchdan) [inline src/web/src/components/ScheduleEvent.vue:10]: These should be removed completely. A comment out is fine, but we either want to display the actual location, or just not have a row.

File: src/web/src/components/ScheduleEvent.vue (hunk 2 above)
Comment (marchdan) [inline src/web/src/components/ScheduleEvent.vue:13]: These should be removed completely. A comment out is fine, but we either want to display the actual location, or just not have a row.

**Comment by marchdan** [comment]
@jshom Thoughts on this? 
Text feels a little small, and idk about the scroll bar.

File: src/web/src/components/ScheduleEvent.vue (hunk 1 above)
Comment (tygentry) [reply src/web/src/components/ScheduleEvent.vue]: Forgot to push latest commit, should be updated on GitHub with them removed now.

File: src/web/src/components/ScheduleEvent.vue (hunk 2 above)
Comment (tygentry) [reply src/web/src/components/ScheduleEvent.vue]: Forgot to push latest commit, should be updated on GitHub with them removed now.

**Comment by jshom** [comment]
looks great on mac/safari,chrome, I think good for merge.
how does it look on windows and various browsers
How does it look on windows
Screen Shot 2020-04-12 at 9 26 36 PM

**Comment by marchdan** [comment]
sched-info
@jshom Here is my current view of it. How does this look? I am not sure how I feel about the scroll bars. The text sizing we don't have much choice over.

**Pull request merged**